### PR TITLE
Contact form pull request

### PR DESCRIPTION
**Pull request by alicemirigo92**
Add html file
Add a contact form
Add style css

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -420,3 +420,103 @@ ul {
...
+  gap: 15px;
+  width: 288px;
+}
+
+form > input {
+  height: 52px;
+  padding-top: 4px;
+  padding-bottom: 33px;
+  padding-left: 16px;
+  border-radius: 8px;
```
Comment (Owusu-Desmond) [inline style.css]: - [x] Please make sure that your project is responsive on a mobile screen. but as you can see from the screenshot below, you have all sections of your project not responsive at a screen width of `375px` :smile: Please kindly fix this by adjusting the style of your sections.
- [x] After fixing this, your sections should cover the mobile screen and there should be no white space left on the right side

Screenshot from 2022-12-07 09-54-40

File: style.css [hunk 2]
```diff
@@ -420,3 +420,103 @@ ul {
...
   border: 1px solid #e2505c;
 }
+
+.form-section {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  grid-gap: 24px;
+  background-image: url(./images/image1.png);
+  background-repeat: no-repeat;
+}
```
Comment (Owusu-Desmond) [inline style.css]: - [x] Please make sure that flexbox is used to lay out the elements in the form section :pray: . But as you can see, you used grid.

File: style.css [hunk 3]
```diff
@@ -420,3 +420,103 @@ ul {
...
+
+#cbn-b {
+  width: 288px;
+  height: 44px;
+  left: 16px;
+  top: 4652px;
+  background: #1a2236;
+  color: #fff;
+}
```
Comment (Owusu-Desmond) [inline style.css]: - [x] Why don't you give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @alicemirigo92 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alicemirigo92** [comment]
@Owusu-Desmond Done

File: style.css [hunk 4]
```diff
@@ -420,3 +420,104 @@ ul {
...
+  background-image: url(./images/image1.png);
+  background-repeat: no-repeat;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 24%;
+}
```
Comment (franclobo) [inline style.css:436]: - [x] As the last code reviewer said please 🙏🏽 make sure that the content covers all space in the mobile screen for a better user interface 👇🏽 

image

File: style.css [hunk 5]
```diff
@@ -420,3 +420,104 @@ ul {
...
+
+#cbn-b {
+  width: 100%;
+  height: 43px;
+  left: 40px;
+  top: 4652px;
+  background: #1a2236;
+  color: #fff;
+  margin-bottom: 40px;
+}
```
Comment (franclobo) [inline style.css:474]: - [x] **[OPTIONAL]** We could use `cursor: pointer;` as the last code reviewer suggested to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) 🙏
- [x] **[OPTIONAL]** Please 🙏🏽 implement the button interactions (enable, hover, etc.).

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @alicemirigo92, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] **Kindly ⚠️ complete all the project requirements and code reviewer suggestions ⚠️ before submitting your project again.**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (alicemirigo92) [reply style.css:474]: @franclobo Done...do i have to resubmit it again for review or you can check it from here?

**Review by Yazino12 — APPROVED** [review]
## Hi @alicemirigo92 🖐🏾,

You have done a great job adding the changes requested by a previous reviewer 🎯, and there is nothing else left to do other than it's time to merge it 🚀🚀.

### To Highlight:
- Correct user of GitHub flow ✔
- Descriptive PR summary ✔

# STATUS: APPROVED ✅✅✅
THERE
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions

- **_[OPTIONAL]_** Kindly keep in mind that this project is a pair-programming project and each group member needs to contribute their code in each repository and then open a separate pull request in their repo 📝.

- **_[OPTIONAL]_** Also keep in mind to use the latest [Microverse Readme Template](https://github.com/microverseinc/readme-template) and mention your pair programming partners there 🙌🏽.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**